products/roundcube.md

```diff
@@ -43,7 +43,7 @@ releases:
     eol: true
     support: false
     
-link: https://roundcube.net/news/2021/10/18/roundcube-1.5.0-released
+link: https://roundcube.net/news/2021/11/28/update-1.5.1-released
```

Comment:
```suggestion
link: https://roundcube.net/news/2021/10/18/roundcube-1.5.0-released
```

Since `link` is used for linking to release policy, we should keep this to 1.5.0 since only the last release announcement has the relevant point about LTS:

>With the release of Roundcube 1.5.0, the previous stable release branches 1.4.x and 1.3.x will change into LTS low maintenance mode which means they will only receive important security updates but no longer any regular improvement updates. The 1.2.x series is no longer supported and maintained.

Or is there a better link we can point to?

Reply:
Changelog already contains similar information. I am reverting this for LTS policy.